feature-libs/customer-ticketing/core/config/default-customer-ticketing-config.ts

```diff
@@ -1,5 +1,20 @@
 import { CustomerTicketingConfig } from './customer-ticketing-config';
 
 export const defaultCustomerTicketingConfig: CustomerTicketingConfig = {
-  customerTicketing: {},
+  customerTicketing: {
```

Comment:
I know you used `asm` as reference most likely, but move this default to the root instead

ASM was part of the first code-splitting that we had done, but was 'outdated' in terms of conventions